server/db.py

```diff
@@ -192,6 +196,35 @@ def clean_db(self):
             self.clear_excess_history(self.utxo_flush_count)
         self.clear_excess_undo_info()
 
+    def fs_update_header_offsets(self, offset_start, height_start, headers):
+        if self.coin.STATIC_BLOCK_HEADERS:
+            return
+        # Write dynamic header offsets, each offset is 64 bits
+        offset = offset_start
+        offsets = []
+        for h in headers:
+            offset += len(h)
+            offsets.append(pack("<Q", offset))
+        self.headers_offsets_file.write(height_start * 8, b''.join(offsets))
+
+    def dynamic_header_offset(self, height):
+        assert not self.coin.STATIC_BLOCK_HEADERS
+        # The offsets are 64 bits in order to accommodate coins with big
+        # headers that will accumulate GBs of header data after some years
+        # We don't store the offset of the genesis header as it's always 0
+        # so this means that the offset for height 1 is stored at
+        # position 0, for height 2 at 8 and so on.
+        if height > 0:
```

Comment:
I'd rather waste 8 bytes on disk and not have the extra comment and if statement.  Any reason not to store height 0 as 0 too?  How is it not getting written?  I might be missing something.

And can you add yourself to AUTHORS?  This 2nd commit looks good otherwise.